h/templates/layouts/base.html

```diff
@@ -5,16 +5,16 @@
       <meta charset="UTF-8" />
       <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
       <meta name="viewport" content="width=device-width,initial-scale=1" />
-      {% for attrs in meta_attrs -%}
```

Comment:
Is that what that cute little dash randomly attached to the side of the % does? Neat! :stuck_out_tongue_winking_eye: 

Sorry...new to Jinja. More familiar with Mustach and Handlebars.

Thanks @aron. Will fix.